sa/type-converter.go

```diff
@@ -38,6 +38,16 @@ func (tc BoulderTypeConverter) ToDb(val interface{}) (interface{}, error) {
 		return string(t), nil
 	case core.OCSPStatus:
 		return string(t), nil
+	case *core.AcmeURL:
```

Comment:
Ah, looks like I was wrong about *core.AcmeURL. It was a *slice* of them that I put in here.

I'm not so sure I should have put in that []*core.AcmeURL and I'm not so sure that we should put this one in, either.